how-to/how-to-template-guide.md

```diff
@@ -0,0 +1,149 @@
+# How-to guide
+
+information_source: Read this document before you start working on the [how-to template](how-to-template.md).
```

Comment:
**nitpick:** The emoji did not render correctly. I believe a colon needs to be added before information_source.

Reply:
Fixed